gdraw/gfilechooser.c

```diff
@@ -36,6 +37,26 @@
 #include "gwidgetP.h"
 #include "ustring.h"
 #include "utype.h"
+#ifdef WIN32
+#include <windows.h>
+
+// Returns "CDQ" if C:\, D:\ and Q:\ are available.
+// This could probably be put in fsys.c, but we don't need it anywhere else,
+// and shouldn't ever.
+static unsigned int WIN32_DRIVES_MAXLEN = 26;
+static char* _DriveLetters() {
+    int idx = -1;
+    char* ret = calloc(WIN32_DRIVES_MAXLEN+1,sizeof(char));
+    DWORD drives = GetLogicalDrives();
+    assert(drives > 0);
```

Comment:
This assert is pointless because DWORD is unsigned

Reply:
Can't it still be zero though?

Reply:
fair